flatpak/org.nickvision.tagger.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id" : "org.nickvision.tagger",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "env": {
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
+        }
+    },
+    "command" : "/app/bin/org.nickvision.tagger",
+    "finish-args":[
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home"
```

Comment:
Any reason for this and not portals?

Reply:
Not in particular...I remember portals us issues with folders as compared to files...as with folder we weren't gettong the correct path returns, or it was still read-only and users couldn't tag any files.